cmd/kind/kind.go

```diff
@@ -67,38 +53,20 @@ func NewCommand() *cobra.Command {
 	return cmd
 }
 
-func runE(flags *Flags, cmd *cobra.Command, args []string) error {
-	level := defaultLevel
-	parsed, err := log.ParseLevel(flags.LogLevel)
-	if err != nil {
-		log.Warnf("Invalid log level '%s', defaulting to '%s'", flags.LogLevel, level)
-	} else {
-		level = parsed
-	}
-	log.SetLevel(level)
-	return nil
-}
-
 // Run runs the `kind` root command
 func Run() error {
 	return NewCommand().Execute()
 }
 
+func init() {
+	//log.InitFlags(nil)
+	flag.CommandLine.AddGoFlagSet(goflag.CommandLine)
+}
+
 // Main wraps Run, adding a log.Fatal(err) on error, and setting the log formatter
 func Main() {
-	// let's explicitly set stdout
-	log.SetOutput(os.Stdout)
-	// this formatter is the default, but the timestamps output aren't
-	// particularly useful, they're relative to the command start
-	log.SetFormatter(&log.TextFormatter{
-		FullTimestamp:   true,
-		TimestampFormat: "15:04:05",
-		// we force colors because this only forces over the isTerminal check
-		// and this will not be accurately checkable later on when we wrap
-		// the logger output with our logutil.StatusFriendlyWriter
-		ForceColors: logutil.IsTerminal(log.StandardLogger().Out),
-	})
 	if err := Run(); err != nil {
```

Comment:
we'll probably have to make this:
```
cmd := NewCommand()
// setup klog
klogFlags := golfag.NewFlagSet("kind", goflag. ContinueOnError)
log.InitFlags(klogFlags)
cmd.PersistentFlags().AddGoFlagSet(klogFlags)
// now we can run
if err := cmd.Execute(); err != nil {
```

Reply:
which is a little unfortunate. at some point ideally we can bind klog flags in NewCommand at the root instead.